packages/extension/src/view/devtools/pages/layout.tsx

```diff
@@ -202,23 +200,111 @@ const Layout = ({ setSidebarData }: LayoutProps) => {
 
   const buttonReloadActionCompnent = useMemo(() => {
     return (
-      <Button text={'Reload'} onClick={handleSettingsChange} variant="large" />
+      <div className="flex items-center gap-5">
+        <Button
+          text="Yes"
+          size="large"
+          onClick={() => {
+            handleSettingsChange();
+          }}
+          variant="success"
+        />
+        <Button
+          text="Cancel"
+          size="large"
+          onClick={async () => {
+            await chrome.storage.session.remove([
+              'isUsingCDP',
+              'pendingReload',
+            ]);
+            setSettingsChanged(false);
+            setIsUsingCDPForSettingsPageDisplay(true);
+          }}
+        />
+      </div>
     );
-  }, [handleSettingsChange]);
+  }, [
+    handleSettingsChange,
+    setIsUsingCDPForSettingsPageDisplay,
+    setSettingsChanged,
+  ]);
+
+  const isUsingCDPCondition = useMemo(() => {
+    if (isUsingCDPForSettingsPageDisplay) {
+      return true;
+    } else {
+      return false;
+    }
+  }, [isUsingCDPForSettingsPageDisplay]);
```

Comment:
```suggestion
const isUsingCDPCondition = useMemo(
  () => isUsingCDPForSettingsPageDisplay,
  [isUsingCDPForSettingsPageDisplay]
);
```

or use `!!isUsingCDPForSettingsPageDisplay` if not boolean.